chapter/chapter.md

```diff
@@ -70,6 +74,9 @@ or *mutated*,
 in place:
 once a value is defined,
 it is *immutable*.
+
+> FIXME I'm assuming this definition is purposefully abridged to reduce clutter?
```

Comment:
yes - anyone who doesn't know what "immutable" is probably won't be able to make much sense of the rest of the chapter